azkaban-web-server/src/main/java/cloudflow/models/ExecutionDetailedResponse.java

```diff
@@ -0,0 +1,22 @@
+package cloudflow.models;
+
+import azkaban.executor.ExecutableFlow;
+import com.sun.istack.internal.NotNull;
+import org.codehaus.jackson.annotate.JsonProperty;
+
+public class ExecutionDetailedResponse extends ExecutionBasicResponse {
+
+  @JsonProperty("rootFlow")
+  @NotNull
```

Comment:
Is this annotation working/validating for you?

Reply:
JsonProperty annotation works as expected. 
For `NotNull` and 'Nullable' the primary purpose was documentation. The code is not referring to the correct version of these annotations though and I have a ticket for fixing this.